### PR TITLE
fix(price-feeder): Fix gate provider getCandlePrices

### PR DESCRIPTION
**Pull request by zarazan**
- Fix the gate providers `getCandlePrices` function to use the correct currency pair string for the stored map of candles
- Updates the price-feeder integration test to use our current sample config
- Removes the noisy websocket send message logger - only log when subscription messages are sent (excludes heartbeat/ping messages)

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by rbajollari — APPROVED** [review]
🔥

File: price-feeder/config/config.go [hunk 1]
```diff
@@ -171,6 +171,14 @@ func (c Config) ProviderPairs() map[provider.Name][]types.CurrencyPair {
 	return providerPairs
 }
 
+func (c Config) ProviderEndpointsMap() map[provider.Name]provider.Endpoint {
```
Comment (robert-zaremba) [inline price-feeder/config/config.go:174]: godoc comment is missing

File: price-feeder/oracle/provider/websocket_controller.go [hunk 2]
```diff
@@ -285,6 +285,9 @@ func (conn *WebsocketConnection) close() {
 
 	conn.logger.Debug().Msg("closing websocket")
```
Comment (robert-zaremba) [inline price-feeder/oracle/provider/websocket_controller.go:286]: shall we have this log after the `conn.client == nil` check?

File: price-feeder/tests/integration/provider_test.go [hunk 3]
```diff
@@ -38,15 +38,18 @@ func (s *IntegrationTestSuite) TestWebsocketProviders() {
...
 		currencyPairs := pairs
+		endpoint := endpoints[providerName]
 		s.T().Run(string(providerName), func(t *testing.T) {
 			t.Parallel()
 			ctx, cancel := context.WithCancel(context.Background())
-			pvd, _ := oracle.NewProvider(ctx, providerName, getLogger(), provider.Endpoint{}, currencyPairs...)
-			time.Sleep(30 * time.Second) // wait for provider to connect and receive some prices
-			checkForPrices(t, pvd, currencyPairs)
+			pvd, _ := oracle.NewProvider(ctx, providerName, getLogger(), endpoint, currencyPairs...)
+			time.Sleep(60 * time.Second) // wait for provider to connect and receive some prices
```
Comment (robert-zaremba) [inline price-feeder/tests/integration/provider_test.go:51]: why increasing to 60?

File: price-feeder/tests/integration/provider_test.go [hunk 4]
```diff
@@ -69,12 +72,12 @@ func (s *IntegrationTestSuite) TestSubscribeCurrencyPairs() {
 
 	time.Sleep(25 * time.Second) // wait for provider to connect and receive some prices
 
-	checkForPrices(s.T(), pvd, currencyPairs)
+	checkForPrices(s.T(), pvd, currencyPairs, "Okx")
```
Comment (robert-zaremba) [inline price-feeder/tests/integration/provider_test.go:75]: should be "OKX"

File: price-feeder/oracle/provider/websocket_controller.go (hunk 2 above)
Comment (zarazan) [reply price-feeder/oracle/provider/websocket_controller.go:286]: When the client is nil it just means the socket has already been closed and there will be a corresponding error message for that event.

File: price-feeder/tests/integration/provider_test.go (hunk 3 above)
Comment (zarazan) [reply price-feeder/tests/integration/provider_test.go:51]: The mexc pairs don't always show up within 30 seconds. This test doesn't run each PR.

File: price-feeder/tests/integration/provider_test.go (hunk 4 above)
Comment (zarazan) [reply price-feeder/tests/integration/provider_test.go]: https://github.com/umee-network/umee/pull/1791/commits/ceeb48c9decee07a966821efeb5285fc8f15afb4

File: price-feeder/config/config.go (hunk 1 above)
Comment (zarazan) [reply price-feeder/config/config.go:176]: https://github.com/umee-network/umee/pull/1791/commits/923039dcdc0e6803d2ddb7bd5b8097df8f541ce9

**Pull request merged**